src/styles/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 40px;
+}
+
+.card__item-name {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0 16px 4px;
+  color: #060b35;
+}
+
+.card__product-code {
+  font-size: 10px;
+  margin: 0 auto 16px;
+  font-weight: 400;
+  color: #616070;
+  line-height: 14px;
+  margin-left: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  align-items: center;
+  line-height: 18px;
+}
+
+.card__star-reviews {
+  padding-right: 16px;
+  font-size: 10px;
+  margin: 0;
+  font-weight: 400;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__price-name {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__price-number {
+  font-size: 16px;
+  font-weight: 600;
+  color: #060b35;
+}
+
+.card__buttom {
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  margin: 0 16px 18px;
+  color: white;
+  text-align: center;
+  width: 166px;
+  border: 1px solid #00acdc;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+}
+
+.card__buttom:hover {
+  background-color: white;
+  color: #00acdc;
+  transition: 0.3s;
```

Comment:
Will be better to specify the properties, for which you use transition (for now it'll use transition for all props, and it can affect on performance in the bad way)